README.md

```diff
@@ -129,4 +142,87 @@ Here is a one page example of using the proxy, assume that this is running on `l
     </script>
 </body>
 </html>
-```
\ No newline at end of file
+```
+
+### Client Credentials Flow (Guest Access) <a name="clientflow"></a>
+
+If you want to allow users to access your app without authorizing Spotify, you can use the client credentials flow. Note that this flow does not require you to define any scopes in the `Reveal Config Vars` section of your deployed app (however, this also means your application can only access publicly available endpoints and data). More information on this flow can be found [here](https://developer.spotify.com/documentation/general/guides/authorization-guide/#client-credentials-flow). 
+
+Create a link that goes to the deployed servers URL and the `/guest` endpoint, where `https://someapp.herokuapp.com` is your URL for the heroku server..
+
+```html
+<a href="https://someapp.herokuapp.com/guest">Continue as guest</a>
+```
+
+This will send the request to Spotify's API token endpoint, which then redirects back to your application with an object containing the guest access token.
+
+Here is a one page example of using the proxy, assume that this is running on `localhost:3400` and that has been added as the `APP_URL` for the proxy on heroku.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Spotify Proxy Test</title>
+</head>
+<body>
+    <!-- The anchor tag to kick off the log in -->
+    <a href="https://spotify-proxy-test.herokuapp.com/guest">Continue as guest</a>
+    <script src='https://code.jquery.com/jquery-3.2.1.min.js' integrity='sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4='crossorigin='anonymous'></script>
+
+    <script>
+        const app = {};
+        //Object to store our tokens
+        app.tokenInfo = {};
+
+        //This method is used to get the token for every request. Since a token only lasts for 3600ms we need to get a new token for each request
```

Comment:
3600 is a strange number for a _milliseconds_ value. Are you sure this isn't 3600 _seconds_ (i.e 60 minutes, i.e. 1 hour)?